src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/commands/ImportCommands.kt

```diff
@@ -22,11 +22,14 @@ class ImportCommands(
 
   @ShellMethod("Imports prices for the given supplier from S3. This command deletes all existing prices for the given supplier.")
   fun importPrices(supplier: Supplier) {
-    importService.importPrices(supplier)
+    when (supplier) {
+      Supplier.UNKNOWN -> throw RuntimeException("UNKNOWN is not a valid supplier")
+      else -> importService.importPrices(supplier)
+    }
   }
 
   /**
-   * Due to potential the large volumes of data each day is imported as an individual import to reduce the memory footprint.
+   * Due to the potential the large volumes of data, each day is imported as an individual import to reduce the memory footprint.
```

Comment:
This might make more sense as "Due to potentially large volumes of data"?

Reply:
Damn it still got it wrong :facepalm: